### PR TITLE
Apply autopep8 for formatting

### PR DESCRIPTION
**Pull request by matiasilva**
Also closes #31

### CODE REVIEW

File: pico_project.py [hunk 1]
```diff
@@ -425,7 +436,8 @@ def __init__(self, parent, initial_config):
     def init_window(self, args):
         self.configure(background=GetBackground())
         self.title("Advanced Configuration")
-        ttk.Label(self, text="Select the advanced options you wish to enable or change. Note that you really should understand the implications of changing these items before using them!").grid(row=0, column=0, columnspan=5)
+        ttk.Label(self, text="Select the advanced options you wish to enable or change. \
+            Note that you really should understand the implications of changing these items before using them!").grid(row=0, column=0, columnspan=5)
```
Comment (lurch) [inline pico_project.py:440]: You need to be careful here :wink: 
```python3
>>> mystr = "this is a string \
...          split over two lines"
>>> mystr
'this is a string          split over two lines'
```

File: pico_project.py [hunk 2]
```diff
@@ -1132,10 +1179,12 @@ def DoEverything(parent, params):
...
     # If there is we abort unless the overwrite flag it set
     if os.path.exists(CMAKELIST_FILENAME):
-        if not params.wantOverwrite :
+        if not params.wantOverwrite:
             if params.wantGUI:
                 # We can ask the user if they want to overwrite
-                y = mb.askquestion('Raspberry Pi Pico Project Generator', 'There already appears to be a project in this folder. \nPress Yes to overwrite project files, or Cancel to chose another folder')
+                y = mb.askquestion(
+                    'Raspberry Pi Pico Project Generator',
+                    'There already appears to be a project in this folder. \nPress Yes to overwrite project files, or Cancel to chose another folder')
```
Comment (lurch) [inline pico_project.py:1187]: Does this also go over your maximum line length? :shrug:

Comment (matiasilva) [reply pico_project.py:1187]: I've set my max line length at 120 chars. I think it does go over in this case

File: pico_project.py [hunk 3]
```diff
@@ -1214,11 +1263,11 @@ def DoEverything(parent, params):
...
 c = CheckPrerequisites()
 
-## TODO Do both warnings in the same error message so user does have to keep coming back to find still more to do
+# TODO Do both warnings in the same error message so user does have to keep coming back to find still more to do
 
 if c == None:
     m = 'Unable to find the `' + COMPILER_NAME + '` compiler\n'
-    m +='You will need to install an appropriate compiler to build a Raspberry Pi Pico project\n'
+    m += 'You will need to install an appropriate compiler to build a Raspberry Pi Pico project\n'
     m += 'See the Raspberry Pi Pico documentation for how to do this on your particular platform\n'
```
Comment (lurch) [inline pico_project.py:1271]: I'll leave it up to you to decide whether it comes within the scope of this PR or not, but note that you _could_ also do this with a triple-quoted string e.g.
```python3
m = '''Unable to find the `{}` compiler
You will need to install an appropriate compiler to build a Raspberry Pi Pico project
See the Raspberry Pi Pico documentation for how to do this on your particular platform
'''.format(COMPILER_NAME)

Comment (matiasilva) [reply pico_project.py:1271]: These strings could potentially be pulled out into a separate file, but yes probably best for another PR.

**Comment by JamesH65** [comment]
Bugger, already merged the template change, think this one should have gone first. I pressume its just a matter of running the pep8 code formatter over the files, so might close this and do it again on the latest tree once the rest of the code has been merged.

**Pull request closed without merging**